docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+# Accessibility
+
+## General Practices
+
+As a general rule, accessible software is achieved by following web development best practices. As Treeherder uses the `reactstrap` library, it is recommended using one of its components, if possible.
+
+If you can not find a suitable component, prefer [HTML5 semantic tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#Semantics_in_HTML) and avoid using generic tags, such as `span` and `div`. Moreover, try to use more **headings** (`h1` to `h6`) whenever possible, because screen readers users usually navigate by headings and that is only possible when developers insert them in the code.
+
+If you can build your layout neither with `reactstrap` nor with semantic tags, please insert `aria-role` in them. There are examples of how to use `aria-role` in: [Tables](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role), [Progress Bar](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role), [List](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role).
+
+`aria-role` is important because it gives meaning to the generic HTML tag. That gives some direction to screen readers on which functionalities to provide and some information to users.
+
+**In summary**, implement your layout trying to use these in the following order:
+
+1. Use `reactstrap`;
+2. Use HTML semantic tags;
+3. Use `aria-role` attributes.
+
+### Reactstrap
+
+Some elements in Treeherder's UI are borrowed from the `reactstrap` library. As a general rule, its elements are already accessible.
+That library is built on top of Bootstrap, therefore the [Bootstrap Accessibility documentation](https://getbootstrap.com/docs/4.0/getting-started/accessibility/) is the one to look for, if in doubt.
+
+There are some issues on **Color Contrast** that are pointed by Bootstrap itself. This was solved in Treeherder by providing [**_custom color classes_**](accessibility.md#colors).
```

Comment:
I don't think this sentence is needed (and it's not really related to reactstrap per say), so it can be removed.